docs/markdown/Release-notes.md

```diff
@@ -1 +1,6 @@
 # Release notes
+
+## Added Wind River Diab compiler suite for bare-metal PowerPC
+
+Tested with C++, C and assembler using [Diab](https://www.windriver.com/products/diab-compiler) 5.2.1.0 from 2004.
```

Comment:
You've mentioned C here, but there's no C compiler implementation, just a C++. That means if a user requests `project('name', 'c')` then this compiler won't be selected.